test/data/noConflict.html

```diff
@@ -0,0 +1,31 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+	<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
+	<script>var QUnit = parent.QUnit</script>
+	<script src="testinit.js"></script>
+	<script src="../../src/sizzle.js"></script>
+	<script>
+		var Sizzle1 = Sizzle;
+	</script>
+	<script src="../../src/sizzle.js"></script>
```

Comment:
These should load from `dist`; see https://github.com/SaptakS/sizzle/blob/master/test/data/mixed_sort.html#L7 .